functionary/core/models/environment.py

```diff
@@ -17,43 +17,36 @@ class Environment(models.Model):
         id: unique identifier (UUID)
         name: the name of the environment
         team: the Team that this environment belongs to
-        default: when true, this environment will act as the default for a Team in
-                 cases where an environment is not specified.
     """
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     name = models.CharField(max_length=64)
     team = models.ForeignKey(
         to="Team", related_name="environments", on_delete=models.CASCADE, db_index=True
     )
-    default = models.BooleanField(default=False)
```

Comment:
Take a look at the file `core/fixtures/bootstrap.yaml`.  It has references to this field that will need to be deleted.  Once that's done, make sure you can successfully run the vscode task "functionary: apply migrations and load fixtures".